api/nodes/serializers.py

```diff
@@ -1540,13 +1555,15 @@ class Meta:
 
 class DraftRegistrationDetailSerializer(DraftRegistrationSerializer):
     """
-    Overrides DraftRegistrationSerializer to make id and registration_metadata required.
+    Overrides DraftRegistrationSerializer to make id required.
     registration_supplement cannot be changed after draft has been created.
 
     Also makes registration_supplement read-only.
+
+    Either pass in registration_metadata (old workflow) or registration_responses
+    (new workflow), not both.  registration_metadata will eventually be deprecated.
     """
     id = IDField(source='_id', required=True)
-    registration_metadata = ser.DictField(required=True)
```

Comment:
Before, we required this field when updating a `DraftRegistration` because it was the only field you *could* update.  Now that you can update either `registration_metadata` or `registration_responses`, it doesn't make sense to make `registration_metadata` required here. (And in fact, we want to move to the new workflow and encourage writing to `registration_responses`!